### ISSUE TITLE
there is no NODE_ENV on .env.example

### ISSUE THREAD

**Issue by HoseinGhanbari** [microsoft/TypeScript-Node-Starter#282]
hey there, thanks for this great node starter
there is currently no `NODE_ENV ` declaration on `.env.example`
e.g. `NODE_ENV=development`

**Comment by peterblazejewicz** [comment]
Hey, thanks for feedback!
Would you llike to see a content like sample `.env` on `dotenv`?
https://github.com/motdotla/dotenv/blob/c1645c96aa8dcd2dbe5f9b086a8164166f2806f2/examples/typescript/.env.example
thx!

**Comment by HoseinGhanbari** [comment]
yeah this is helpful, as people like me might try `cross-env`, `env` or `dotenv` BUT everything is working out of the box in the starter; of course as I couldn't find the `NODE_ENV` declaration, I forgot to set it directly on `.env`.

Thanks!
